### DISCUSSION TITLE
Arming not working

### DISCUSSION THREAD

**Discussion by IsaacSafeSight** [iNavFlight/inav#7381] [Q&A]
Hello Im trying to attach a Matek optical flow board to my Rekon 5 and receiving this error when trying to arm NAV HWFAIL CLI and info would be greatly appreciated!

**Comment by DzikuVx** [comment]
https://github.com/iNavFlight/inav/wiki/%22Something%22-is-disabled----Reasons

czw., 26 sie 2021, 21:21 użytkownik IsaacSafeSight ***@***.***>
napisał:

**Reply by IsaacSafeSight** [reply]
Thank you so much! Do you have any experience attaching a Matek optical flow board to an FPV drone? Im struggling to get the hardware working. Thank you so much!
